source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -650,6 +627,7 @@ ProcessingRequest Filter::setupBodyChunk(ProcessorState& state, const Buffer::In
                                          bool end_stream) {
   ENVOY_LOG(debug, "Sending a body chunk of {} bytes, end_stream {}", data.length(), end_stream);
   ProcessingRequest req;
+  addAttributes(state, req);
```

Comment:
Dont we need the same for the response bodies as well? or is this function called in the response body path as well?

Reply:
This is called from the `onData` helper method which is called on both decode and encode